package.json

```diff
@@ -48,47 +50,22 @@
     "homepage": "https://github.com/demergent-labs/azle#readme",
     "dependencies": {
         "@dfinity/agent": "^2.4.1",
-        "@dfinity/identity-secp256k1": "^2.4.1",
-        "@sinonjs/text-encoding": "^0.7.3",
         "binaryen": "^116.0.0",
-        "buffer": "^6.0.3",
-        "chokidar": "^4.0.3",
-        "class-transformer": "^0.5.1",
-        "class-validator": "^0.14.1",
-        "crypto-browserify": "^3.12.0",
         "cuzz": "^0.0.6",
-        "deep-is": "^0.1.4",
         "esbuild": "^0.25.2",
-        "esbuild-plugin-tsc": "^0.5.0",
-        "ethers": "^6.13.5",
-        "fs-extra": "^11.3.0",
-        "glob": "^11.0.1",
-        "hash-of-directory": "^1.0.1",
-        "http-message-parser": "^0.0.34",
-        "intl": "^1.2.5",
-        "js-sha256": "0.11.0",
         "jssha": "^3.3.1",
-        "net": "^1.0.2",
-        "pako": "^2.1.0",
-        "reflect-metadata": "^0.2.2",
-        "repl": "^0.1.3",
         "tsx": "^4.19.3",
-        "ts-jest-mock-import-meta": "^1.3.0",
         "typescript": "^5.8.3",
-        "uuid": "^11.1.0",
-        "wasmedge_quickjs": "github:demergent-labs/wasmedge-quickjs#3b3b0ee91248ccf9cd954ffafbac7e024648af92"
+        "uuid": "^11.1.0"
     },
     "devDependencies": {
         "@dfinity/identity": "^2.4.1",
         "@eslint/js": "^9.24.0",
-        "@types/deep-equal": "^1.0.4",
-        "@types/fs-extra": "11.0.4",
-        "@types/pako": "^2.0.3",
         "@types/semver": "^7.7.0",
         "@types/uuid": "^10.0.0",
         "@typescript-eslint/eslint-plugin": "^8.30.1",
         "@typescript-eslint/parser": "^8.30.1",
-        "deep-equal": "^2.2.3",
+        "azle-experimental-deps": "github:demergent-labs/azle-experimental-deps#a8ab3214c9a3599ea540d0f429fe0d4b778417cb",
         "eslint": "^9.24.0",
         "eslint-config-prettier": "^10.1.2",
```

Comment:
Remove the `version` from the `azle-experimental-deps` package.json

Reply:
Done